torch/csrc/jit/frontend/schema_matching.cpp

```diff
@@ -678,6 +680,17 @@ Value* emitBuiltinCall(
       schemas.push_back(&op->schema());
   }
 
+  // we might have seen old historic
+  // ops that are deprecated
+  if (variants.size() == 0) {
+    auto oldSchemas =
+        loadPossibleHistoricOps(name.toQualString(), graph_version);
+    for (const auto& old_schema_entry : oldSchemas) {
```

Comment:
Nit:
reserve space for `upgrader_schemas`

Reply:
done